rust/osm2lanes/src/transform.rs

```diff
@@ -145,14 +192,14 @@ fn bus(
     forward_side: &mut [LaneSpec],
     backward_side: &mut [LaneSpec],
 ) {
-    let fwd_bus_spec = if let Some(s) = tags.get("bus:lanes:forward") {
+    let fwd_bus_spec = if let Some(s) = tags.key_get(TagKey::from("bus") + "lanes" + "forward") {
```

Comment:
I'm not sure I understand the problem being addressed? I was kind of expecting some abstraction that handles separators in the values (like `turn:lanes=none|none|through;slight_right`). I guess some keys do get complex, like `cycleway:left:width`. But then I'd expect there to be something breaking apart this string into pieces.

Reply:
Ah, from the PR description, maybe I follow a bit better.. are you just trying to conveniently handle a short vector of strings and deal with `&'static str` vs `String`?

Reply:
yeah, the idea came from the other PR where I needed the `:left:` and `:right:` parts to be programmatic.
I am not convinced though, so I'll leave it on ice until I can better justify needing the complexity :/